pkg/registry/servicecatalog/rest/storage_servicecatalog_test.go

```diff
@@ -82,7 +82,7 @@ func TestV1Alpha1Storage(t *testing.T) {
 	}
 	// TODO: do stuff with broker status storage
 
-	_, serviceClassStorageExists := storageMap["serviceclasses"]
+	_, serviceClassStorageExists := storageMap["clusterserviceclasses"]
 	if !serviceClassStorageExists {
 		t.Fatalf("no service class storage found")
 	}
```

Comment:
Looks like we should add coverage for serviceplans here (in a follow-up).

Reply:
ack.